package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@itwin/itwinui-react",
-  "version": "1.1.0",
+  "version": "1.1.1",
```

Comment:
I can't decide if this should be 1.2.0 or 1.1.1

The `popoverProps` thing is technically an addition but it's not really a feature, so I went with 1.1.1

Reply:
I would say it should be `1.2.0`.